src/Views/LanguageView.vala

```diff
@@ -229,6 +236,37 @@ public class Installer.LanguageView : AbstractInstallerView {
         next_step ();
     }
 
+    private void on_next_button_secondary_clicked () {
+        var dialog = new Granite.MessageDialog.with_image_from_icon_name (
+            _("Unexpected click"),
+            _("You clicked the right mouse button where a left-click was expected. Do you want to switch left and right mouse buttons?"),
```

Comment:
I think the way this is phrased kind of makes left-handed usage sound non-optimal and is maybe a bit micro-aggression-y. We can probably use more precise and objective language here to be more inclusive and clarify the question as well.

```suggestion
            _("Select which mouse button to use for primary-click"),
            _("The right mouse button was used where a primary-click was expected. You can choose to use the left or right mouse button as the primary-click button."),
```

Any thoughts on that @elementary/ux?

Reply:
We could leave it more open-ended like this, but I'd also be okay guiding more towards switching the buttons since this is only shown in response to a right click. Trying to stick to the HIG convention of:

>**Suggestion**
>More info and implications

```suggestion
            _("Use the right mouse button for primary click?"),
            _("The right mouse button was used where a primary click was expected. You can choose to always use the right mouse button for primary click."),
```

I also considered adding "This can be changed later in System Settings → Mouse & Touchpad → Clicking" but that got real wordy.

Reply:
Oh huh yeah I mean, that might be good to include here. As long as the primary text is clear, I don't think it would be a huge issue to provide more info here